pkg/parse_hal/hal_to_clouddriver_test.go

```diff
@@ -148,7 +148,7 @@ func TestHalToClouddriver(t *testing.T) {
 		t.Run(tt.n, func(t *testing.T) {
 			gotC := parse_hal.HalToClouddriver(tt.h)
 			if !reflect.DeepEqual(gotC, tt.wantC) {
-				t.Errorf("Expected hal config to generate %v, got %v", tt.wantC, gotC)
+				t.Errorf("%s: Expected hal config to generate %v, got %v", tt.n, tt.wantC, gotC)
```

Comment:
I thought I remembered the error output including the name of the test right above each error (which is why you have to pass some name to `t.Run`, but if this makes it even more readable then I'm on board!

Reply:
Great point! I somehow completely missed that in the output, but now that I see the name is there I agree it makes more sense to leave it out of the actual error.  Thanks for pointing that out, I'm still getting used to what this test output looks like!